src/Files.App/Helpers/AppUpdater.cs

```diff
@@ -1,101 +0,0 @@
-using Files.App.Extensions;
```

Comment:
Was this removed by mistake?

Reply:
@yaichenbaum It was unused and replaced by a service implementation.